docs/src/create-apps/app-reference.md

```diff
@@ -83,7 +84,7 @@ To do so, set `size` to one of the following values:
 
 The total resources allocated across all apps and services can't exceed what's in your plan.
 
-### Sizes in development environments
+### Sizes in non-proudction environments
```

Comment:
📋 

```suggestion
### Sizes in non-production environments
```

Reply:
Ah, just noticed this before coming and seeing your comment. Should have caught it sooner :see_no_evil: